src/__stories__/sheet-story.tsx

```diff
@@ -42,6 +42,8 @@ export const Default: StoryComponent = () => {
             <Stack space={16}>
                 <ButtonPrimary
                     disabled={open}
+                    aria-expanded={open}
```

Comment:
you may want to change playroom sheet snippets too

Reply:
done :)